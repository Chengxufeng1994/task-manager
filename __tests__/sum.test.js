const sum = require('../src/utils/sum');

describe('sum test', () => {
  test('should return 5 when a = 2, b = 3', () => {
    const result = sum(2, 3);
    expect(result).toBe(5);
    expect(result).toMatchSnapshot();
  });
});
