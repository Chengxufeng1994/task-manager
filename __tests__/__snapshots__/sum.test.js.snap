// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sum test should return 5 when a = 2, b = 3 1`] = `5`;
